project/game/riddlemaster_view.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, scene, cast, props, script):
                 padding=(10, 10, 10, 10)
             )
         )
-        self.text = arcade.gui.UIInputText(x=500, y=400, width=200, height=50, text="Type Here", text_color=(0,0,0,255))
+        self.text = arcade.gui.UIInputText(x=500, y=400, width=200, height=50, text=" ", text_color=(0,0,0,255))
```

Comment:
by deleting the "type here" it is faster to answer because the player doesn't have to delete it before answering. I have been trying to figure out how to delete the text when the answer box is clicked on, but haven't figured it out yet.